.gitignore

```diff
@@ -0,0 +1,6 @@
+_site
```

Comment:
As we build the site locally by running jekyll.

```
bundler exec jekyll serve
```

The site and metadata is built. We need to ignore these.